benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -511,21 +496,19 @@ msgstr ""
 msgid "core.pages.index.continue"
 msgstr "Comencemos"
 
-#: benefits/core/views.py:59
-msgid "core.pages.agency_index.button.label"
-msgstr "¡Conecta tu tarjeta bancaria a tu descuento hoy!"
-
-#: benefits/core/views.py:62
+#: benefits/core/views.py:61
 msgid "core.pages.agency_index.title"
 msgstr "Introducción"
 
-#: benefits/core/views.py:63
-msgid "core.pages.agency_index.content_title"
+#: benefits/core/views.py:62
+#, fuzzy
+#| msgid "core.pages.agency_index.content_title"
```

Comment:
@angela-tran I ran the make messages command and it added this `fuzzy` line here. What does that mean?

Reply:
Also, it only does fuzzy-guessing the first time that it generates the entry. Once you provide a `msgstr`, it won't try to guess the value again.